src/utils.cpp

```diff
@@ -635,10 +636,13 @@ int gaiaVersion(const QDataStream& stream) {
   if (qtVersion == QDataStream::Qt_4_2) return Gaia_2_0;
   if (qtVersion == QDataStream::Qt_4_3) return Gaia_2_1;
   if (qtVersion == QDataStream::Qt_4_4) return Gaia_2_3;
-
+#ifdef GAIA_QT5
+  if (qtVersion == QDataStream::Qt_5_6) return Gaia_2_4;
```

Comment:
Should add the rest of new Qt versions  up to 5.9?

Reply:
Looked up `Qt` documentation and `QDataStream` version id for `Qt_5_6` is the same for `Qt 5.6` thru `Qt 5.9`.  Which means `Ubuntu 16.04+` can benefit from these changes, if they don't cause issues with existing `sig` files, using `Qt` from `launchpad`.

`Ubuntu 12.04` uses `Qt 5.0` and `Ubuntu 14.04` uses `Qt 5.2` and are likely happy without change.